### ISSUE TITLE
Server protocol version error

### ISSUE THREAD

**Issue by T0RM0R** [ammaraskar/pyCraft#240]
Hello
So Im making a bot script, but getting a server versionmismatch error saying that "Server protocol 754 is not supported" when I try to join a 1.16.5 server. When i look in pycraft-master/minecraft/__init__.py it says that protocol version 754 is supported and if i use start.py to connect to the server it works fine. My question being am I supposed to send some packets or set some option before joining so that the protocol version is correct? 

Ty in advance

**Comment by joodicator** [comment]
Actually, this appears to be an error in the version metadata:
https://github.com/ammaraskar/pyCraft/blob/33293a239517bc4f51fa797c49a95a3c55383482/minecraft/__init__.py#L459-L465
The record for 1.16.4 is marked as supported, but the one for 1.16.5 is not. I think you should now find this to be fixed in bcd156e. Thank you for the report.

**Comment by T0RM0R** [comment]
Hello
Ty for the quick response

I tried the updated version and it still kept showing the error, but turns out that if I move the bot script out of a subfolder made in the pyCraft-master folder it starts working like expected and doesn't give me the error. I dont know what causes it, but moving it out of the subfolder seems to fix it. I also tried it with the start.py script that comes by default and it started showing the same error when moved into a subfolder.
